src/fundus/publishers/my/malay_mail.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class MalayMailParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div.article-body > p")
```

Comment:
Longer articles also contain subheadlines. For example this one: https://www.malaymail.com/news/world/2024/05/02/bogota-cuts-ties-with-israel-over-genocidal-gaza-campaign/131974 
Could you also add a subheadline_selector that covers this case?